odoo/addons/base/controllers/avatar.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from hashlib import sha1
+
+from odoo.http import Controller, request, route, Response
+
+# show avatar if no image or showAvatar 1
+# show image if image
+
+class Avatar(Controller):
+    @route(['/web/avatar/<int:user_id>', '/web/avatar/<int:user_id>/<string:image_field>'], methods=['GET'])
```

Comment:
what is `image_field` used for?

Reply:
What is the `auth` method for this route? It [should be specified explicitly](https://www.odoo.com/documentation/14.0/reference/http.html#routing) and most likely be `'user'` (otherwise the `browse()` in the method is likely to crash.

Reply:
oh, `image_field` was being used before because I used the API to redirect to the image stored in the database, but now that's being done in the widget. Nice catch. I will remove this and add the `auth` method :)